src/routes/birthday/create/index.ts

```diff
@@ -74,17 +43,16 @@ export class BirthdayCreateRoute extends Route {
             });
         } catch (error: any) {
             if (error.name === 'SequelizeUniqueConstraintError') {
-                response.statusMessage = 'User already has a birthday registered';
-                return response
-                    .status(200)
-                    .json({ success: false, error: { code: APIErrorCode.DUPLICATE_ENTRY, message: 'User already has a birthday registered' } });
+                return response.badRequest({
```

Comment:
its not a bad request is it? not sure about the logic, but it can be that there is a already a birthday registered but this makes the request itself not bad.
Wouldn't it be better to return a 200 with the body `success: false` than a `badRequest`?

Reply:
The route defines that we create a guild and if it already exists we can't create it so it's a mistake, right?